src/app/main/app-routing.module.ts

```diff
@@ -9,9 +9,13 @@ import { ConnectComponent } from '../connect/connect.component';
 import { GetComponent } from './get.component';
 import { SettingsModule } from '../settings/settings.module'

```

Comment:
These three modules seems not be used. Can we remove them? 

import { SettingsModule } from '../settings/settings.module'
import { WebServerModule } from '../webserver/webserver.module'
import { WACModule } from "runtime/wac/components/wac.module";

Reply:
Ah, these are actually being used. I was confused because VS CODE shows them in gray color.